setup.py

```diff
@@ -62,10 +62,11 @@
         'test': [
             'check-manifest',
             'mock==1.3.0',
-            'pep8>=1.6.2',
+            'pycodestyle>=2.0.0',
+            'pydocstyle>=1.0.0',
             'pylint>=1.4.3',
```

Comment:
Maybe also bump the pylint version since we probably aren't sure whether everything works on older versions?